test/integration/connection-monitoring-and-pooling/connection_monitoring_and_pooling.spec.test.ts

```diff
@@ -0,0 +1,93 @@
+import { HostAddress, MongoClient } from '../../../src';
+import { shuffle } from '../../../src/utils';
+import { loadSpecTests } from '../../spec';
+import { CmapTest, runCmapTest, ThreadContext } from '../../tools/cmap_spec_runner';
+import { isAnyRequirementSatisfied } from '../../tools/unified-spec-runner/unified-utils';
+
+// These tests rely on a simple "pool.clear()" command, which is not sufficient
+// to properly clear the pool in LB mode, since it requires a serviceId to be passed in
+const LB_SKIP_TESTS = [
+  'must destroy checked in connection if it is stale',
+  'must destroy and must not check out a stale connection if found while iterating available connections'
+];
+
+describe('Connection Monitoring and Pooling Spec Tests (Integration)', function () {
+  const suites: CmapTest[] = loadSpecTests('connection-monitoring-and-pooling');
+
+  for (const test of suites.filter(test => {
+    // TODO(NODE-2993): unskip integration tests for maxConnecting
+    return test.style === 'unit';
+  })) {
+    describe(test.description, function () {
+      let hostAddress: HostAddress, threadContext: ThreadContext, client: MongoClient;
+
+      beforeEach(async function () {
+        let utilClient: MongoClient;
+        if (this.configuration.isLoadBalanced) {
+          if (LB_SKIP_TESTS.some(testDescription => testDescription === test.description)) {
```

Comment:
No, because this isn't "broken". In load balancer mode, the clear method is not supposed to be used, instead another method that clears the pool based on the serviceId must be used. We just overloaded our clear method to do both, so it handles the load balance serviceId based clear if a serviceId is passed in, and it handles the regular clear if no serviceId is passed in. These are unit tests, originally, so they reference the regular clear method, which does not apply in LB mode. (See the clarification in this spec commit: https://github.com/mongodb/specifications/commit/b4423500b018352227b7df8d21b36681db0ad58a)